staging/src/k8s.io/kube-aggregator/pkg/apiserver/metrics.go

```diff
@@ -64,7 +64,7 @@ var (
 		&metrics.HistogramOpts{
 			Name:           "extension_apiserver_request_duration_seconds",
 			Help:           "extension apiserver request duration in seconds broken out by result.",
-			Buckets:        metrics.ExponentialBuckets(0.001, 2, 15),
+			Buckets:        []float64{0.005, 0.025, 0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 1.25, 1.5, 2, 3, 4, 5, 6, 8, 10, 15, 20, 30, 45, 60},
```

Comment:
@benluddy @logicalhan ptal.

Reply:
I don't think we need that much precision for the extension apiserver. The following buckets could be enough to start with and we can adjust then later on if needed.
```
[]float64{0.005, 0.025, 0.05, 0.1, 0.2, 0.5, 1, 2, 5, 10, 25, 60}
```

Reply:
Would it be more useful operationally to spend these timeseries on being able to distinguish between backends? With one or more remote aggregated apiservers, plus an apiextensions-apiserver, what shape will the distribution of request latencies have?

Reply:
Oh. When I first looked at the PR, I assumed it was distinguishing between backends via a `resource` label, but that's not the case. I agree with you that the metric is not very useful if we don't have that.

Reply:
> distinguishing between backends via a resource label

If that's not practical here, we should at least be able to distinguish by group/version since that is available to the APIService controller when it instantiates a handlerProxy.